cli/formatter/formatter.go

```diff
@@ -0,0 +1,778 @@
+package formatter
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/formatter/table"
+	"github.com/tarantool/tt/cli/formatter/text"
+	"gopkg.in/yaml.v2"
+)
+
+// Opts contains formatting options.
+type Opts struct {
+	TransposeMode bool
+	NoGraphics    bool
+	ColWidthMax   int
+	TableDialect  TableDialect
+}
+
+// TableDialect defines a set of supported table dialect.
+type TableDialect int
```

Comment:
We could move the type into a separate file.

Reply:
Done